src/main/cpp/subsystems/ControlBoard.cpp

```diff
@@ -5,27 +5,84 @@
 #include "subsystems/ControlBoard.h"
 #include "CONSTANTS.h"
 
+
 ControlBoard::ControlBoard() {
+  _flywheelDesired = IsDown(_flywheelButton, FLYWHEEL_BUTTON_PORT);
+	_flywheelPrepDesired = IsDown(_flywheelPrepButton, FLYWHEEL_PREP_BUTTON_PORT);
   // Implementation of subsystem construcr goes here.
 }
 
+// double ControlBoard::GetLeftX() {
+//   return _xbox.GetLeftX();
+// }
+// double ControlBoard::GetRightX() {
+//   return _xbox.GetRightX();
+// }
+// double ControlBoard::GetLeftY() {
+//   return _xbox.GetLeftY();
+// }
+// double ControlBoard::GetRightY() {
+//   return _xbox.GetRightY();
+// }
+
+// bool ControlBoard::GetLeftBumper() {
+//   return _xbox.GetLeftBumper();
+// }
+
+// bool ControlBoard::GetLeftTrigger() {
+//   return _xbox.GetLeftTriggerAxis() > LEFT_TRIGGER_BARRIER;
+// }
+
+// bool ControlBoard::GetRightTrigger(){
+//   return _xbox.GetRightTriggerAxis() > RIGHT_TRIGGER_BARRIER;
+// }
+
 double ControlBoard::GetLeftX() {
-  return _xbox.GetLeftX();
+  return _leftJoy.GetX();
 }
 double ControlBoard::GetRightX() {
-  return _xbox.GetRightX();
+  return _rightJoy.GetX();
 }
 double ControlBoard::GetLeftY() {
-  return _xbox.GetLeftY();
+  return _leftJoy.GetY();
 }
 double ControlBoard::GetRightY() {
-  return _xbox.GetRightY();
+  return _rightJoy.GetY();
+}
+
+// bool ControlBoard::GetLeftBumper() {
+//   return _xbox.GetLeftBumper();
+// }
+
+// bool ControlBoard::GetLeftTrigger() {
+//   return _xbox.GetLeftTriggerAxis() > LEFT_TRIGGER_BARRIER;
+// }
+
+// bool ControlBoard::GetRightTrigger(){
+//   return _xbox.GetRightTriggerAxis() > RIGHT_TRIGGER_BARRIER;
+// }
+
+
+// bool ControlBoard::GetX(){
+//   return _flywheelJoyX.GetX();
+// }
+
+bool ControlBoard::IsDown(frc::Joystick &joystick, int PORT){
+  _operatorJoyB.GetRawButton(PORT);
 }
 
-bool ControlBoard::GetLeftBumper() {
-  return _xbox.GetLeftBumper();
+
+bool ControlBoard::GetFlywheelDesired(){
+  return _flywheelDesired;
+}
+
+bool ControlBoard::GetFlywheelPrepDesired(){
+  return _flywheelPrepDesired;
+}
+
+
+void ControlBoard::Update(){
+  _flywheelDesired = IsDown(_flywheelButton, FLYWHEEL_BUTTON_PORT);
+	_flywheelPrepDesired = IsDown(_flywheelPrepButton, FLYWHEEL_PREP_BUTTON_PORT);
 }
```

Comment:
We're moving away from constantly reading/keeping track of the states of the buttons. Check this out: https://docs.wpilib.org/en/stable/docs/software/commandbased/binding-commands-to-triggers.html 

ControlBoard can keep all the buttons in one place. Then, to schedule commands, we'll probably have to pass a reference of a button into RobotContainer..? 🤷 Still haven't thought about restructuring this a whole lot yet.

Also woah @rwlee, the custom triggers are so cool.